core/vm/runtime/env.go

```diff
@@ -39,18 +39,17 @@ func NewEnv(cfg *Config) *vm.EVM {
 		BlobFeeCap: cfg.BlobFeeCap,
 	}
 	blockContext := vm.BlockContext{
-		CanTransfer:       core.CanTransfer,
-		CanTransferMC:     core.CanTransferMC,
-		Transfer:          core.Transfer,
-		TransferMultiCoin: core.TransferMultiCoin,
-		GetHash:           cfg.GetHashFn,
-		Coinbase:          cfg.Coinbase,
-		BlockNumber:       cfg.BlockNumber,
-		Time:              cfg.Time,
-		Difficulty:        cfg.Difficulty,
-		GasLimit:          cfg.GasLimit,
-		BaseFee:           cfg.BaseFee,
-		BlobBaseFee:       cfg.BlobBaseFee,
+		CanTransfer: core.CanTransfer,
+		Transfer:    core.Transfer,
+		GetHash:     cfg.GetHashFn,
+		Coinbase:    cfg.Coinbase,
+		BlockNumber: cfg.BlockNumber,
+		Time:        cfg.Time,
+		Difficulty:  cfg.Difficulty,
+		GasLimit:    cfg.GasLimit,
+		BaseFee:     cfg.BaseFee,
+		BlobBaseFee: cfg.BlobBaseFee,
+		Random:      cfg.Random,
```

Comment:
Why was Random not set in the master branch? 🤔

Reply:
I think we deleted random in the master branch.

Reply:
Indeed, and this addition makes it the same as in geth, thanks for clarifying! 👍